Docs 9532 partner manager endpoints doc edits esb (#98)
mulesoft/docs-partner-manager: modules/ROOT/pages/endpoint-https-receive.adoc

|===

> |Name |Description |Required

Reviewer: If the names are syntax, should we format them as ` `?

Author: Leaving in plain text

|Endpoint Name

> |The name used to identify the endpoint. The `Endpoint Name` field is generated automatically, but can be modified manually.

Reviewer: Is the field generated automatically or the value of that field?
This is a syntax field, not a labeled UI field, right?

Author: This is a field on a datasheet - I have changed it to bold.

|Yes

| Owner

> | The trading partner or host that owns the endpoint. The value in this field is populated automatically and cannot be modified.

Reviewer 2: Please remove: "The value in this field is populated automatically and cannot be modified."

Author: Done

| Yes

| Protocol

> | The protocol selected in the `Protocol` field of the *New endpoint: Receive from partners* window. The value in this field cannot be modified.

Reviewer: Labeled UI field would be * *, not ` `.

See "font choices" in the style ref.

Author: Done in all 3 topics

| Yes

| Description

> | The user-defined field for describing the purpose and use of the endpoint.

Reviewer: Again, are we talking about FIELDS or VALUES within fields?

Author: value!

| No

|URL

> a|The HTTP or HTTPS address where you will receive outbound source messages from backend systems.

Reviewer: ```suggestion
a|The HTTP or HTTPS address that receives outbound source messages from backend systems.
```

> When deploying to CloudHub, the URL is automatically generated when you deploy a message flow that uses this endpoint. To view the URL, you can select the receiving endpoint from your deployed message flow.

Reviewer: ditto re. misplaced modifier (see earlier edit, same text)

Author: Done!

> If you are deploying to on-premises Mule server with a firewall and load balancer, work with your hosting or network security team to obtain the externally accessible DNS or name of the host where the Mule server is running, and replace `{runtime_host}` in the URL with the DNS name of your host. For example, if your DNS host name is `b2b.mycompany.com`, the receiver URL is `+https://b2b.mycompany.com/base-path/+`.

Reviewer: See previous edit, same text.

If you decide to keep "server" instead of replacing it with "instance," be sure to include an article: "...deploying to an on-premises Mule server...."

Author: "Instance" works better, I think!

|

|Connection idle timeout

> |The connection idle timeout period in milliseconds. The value in this field is set to `3000` by default.

Reviewer 2: Please change the default value to 30000.

Reviewer: ^

Author: Done

|

|Base path

> |The base path for the HTTP listener

Reviewer: Please check this table column's punctuation GLOBALLY in the PR.

As with a list, if even one entry contains a complete sentence (ending with a period), then all entries must end with a period.

If you can write around situations like this, that is great!

Peer list entries, steps, and table column entries should be parallel, if possible.

Author: Done!

|Yes

|Basic authentication

> a|When Basic authentication is enabled, specify the user name and password to protect the HTTP listener.

Reviewer 2: "Selected" is fine here. It refers to the choice user makes in the previous field "Authentication type".

Author: Done

> After the configuration or creation of the endpoint is saved, you cannot retrieve or modify the password.

Reviewer: configuration or creation is saved?
Hmm...can we say only "After your new or [re?]configured endpoint is saved, you cannot subsequently..."?

Author: Done!

The backend applications that send JSON or XML payloads to the HTTP receive endpoint must pass the credentials in the HTTP request header.
|

|API Key

> a|The API Key for the HTTP listener.

Reviewer: Why is "key" capitalized? GLOBAL

Author: Fixed!

> When the API Key is selected, specify the HTTP header name and API key to protect the HTTP listener. After the configuration or creation of the endpoint is saved, you cannot retrieve or modify the API key.

Reviewer: passive: "is selected" by whom or what, and when?

Reviewer: ditto re "configuration or creation"

Author: Done!

> The backend applications that send JSON or XML payloads to the HTTP receive endpoint must pass the API key in the HTTP request with the configured HTTP header name.

Reviewer: Note lowercase "key"

Author: Noted and reviewed globally!
